### PR TITLE
Put the add meta action back in

### LINKED ISSUE #53: Need to be able to add a post
This was removed in 7efcee1b4629e75a1dd715b10d449a55cd940156.

### PR DESCRIPTION
**Pull request by voodoochild**
Fixes #53

### CODE REVIEW

File: src/pages/content-edit/nav.js [hunk 1]
```diff
@@ -53,6 +53,17 @@ module.exports = {
         });
 
         // Event handlers
+        ctrl.add = function() {
+            var result;
+
+            result = db.child("content/" + ctrl.schema.key).push({
```
Comment (tivac) [inline src/pages/content-edit/nav.js]: I really need to build a simple data-layer abstraction, this sort of thing living in controllers seems SO dangerous :(

File: src/pages/content-edit/nav.css [hunk 2]
```diff
@@ -198,3 +198,19 @@
     padding: 0;
     overflow: visible;
 }
+
+/* Metadata controls */
+.metas {
+    composes: head from layout_css;
+}
+
+.meta {
```
Comment (tivac) [inline src/pages/content-edit/nav.css]: ?

File: src/pages/content-edit/nav.css [hunk 3]
```diff
@@ -198,3 +198,19 @@
...
+}
+
+.addMeta {
```
Comment (tivac) [inline src/pages/content-edit/nav.css]: ??

File: src/pages/content-edit/nav.css [hunk 4]
```diff
@@ -2,7 +2,7 @@
 @value layout_css: "../layout/layout.css";
 @value anchor: #446;
 @value iconTransition: color 0.2s ease-in;
-@value colorRemove from "../../buttons.css";
+@value buttons_css: "../../buttons.css";
```
Comment (tivac) [inline src/pages/content-edit/nav.css]: Would you move this up w/ `layout_css` please so they can buddies united in doing the same thing?

File: src/pages/content-edit/nav.js (hunk 1 above)
Comment (voodoochild) [inline src/pages/content-edit/nav.js]: :+1:

**Pull request merged**